SQLQuery1.sql

```diff
@@ -0,0 +1,212 @@
+﻿--Tạo bảng
+Create database bang_test
+use bang_test;
+create table Project_user (
+id bigint not null Identity(1,1) primary key,
```

Comment:
khi là khoá chính mình sẽ dùng kiểu này nhá:
 id bigint NOT NULL AUTO_INCREMENT,